src/main/java/com/ebay/myriad/scheduler/TaskFactory.java

```diff
@@ -63,6 +58,31 @@ private static String getFileName(String uri) {
             }
         }
 
+        private CommandInfo getCommandInfo() {
+            MyriadExecutorConfiguration myriadExecutorConfiguration = cfg.getMyriadExecutorConfiguration();
+            String cmdPrefix = "export CAPSULE_CACHE_DIR=`pwd`;echo $CAPSULE_CACHE_DIR; java -Dcapsule.log=verbose -jar ";
+            CommandInfo.Builder commandInfo = CommandInfo.newBuilder();
+            if (myriadExecutorConfiguration.getNodeManagerUri().isPresent() &&
+                    myriadExecutorConfiguration.getCommand().isPresent() &&
+                    myriadExecutorConfiguration.isRemoteDistribution()) {
+                String nmURI = myriadExecutorConfiguration.getNodeManagerUri().get();
+                URI executorURI = URI.newBuilder().setValue(nmURI).setExtract(true).build();
+                //@Todo: determine if "root" or "" is the proper default, to keep backward compatiblility
+                //keeping root for now
```

Comment:
This was an attempt to be as unobtrusive as possible, it was set as root previously.  I like the sentiment of where you're going, but unfortunately container-executor needs a group sticky bit and owned by root, which means we need to have a user running with sudo change the permission, but we may not want to leave that user to run nodemanager.  We could drop this after addressing https://issues.apache.org/jira/browse/MESOS-1790 or if you've got a better way.  I'm very open to suggestions here.  For now defaulting to the current user may be better than root.